.github/workflows/docs-update.yml

```diff
@@ -33,4 +33,4 @@ jobs:
           git push -u origin HEAD:docs-$GITHUB_REPOSITORY-$GITHUB_SHA
           gh pr create -t "Docs Update from $GITHUB_REPOSITORY" -b "This is an automated pull request." -B main -H docs-$GITHUB_REPOSITORY-$GITHUB_SHA
           echo "done"
-          
\ No newline at end of file
+          
```

Comment:
Nice job with the PR! Does anyone know why this new line was added? @Aneurysm9

Reply:
> POSIX defines a text file as a file that contains characters organized into zero or more lines,[[4]](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_403) where lines are sequences of zero or more non-newline characters plus a terminating newline character,[[5]](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) normally LF. 
[Source](https://en.wikipedia.org/wiki/Text_file#Unix_text_files)

It looks to me like this file previously did not have a terminating newline character on the last line.